docs/diagrams/PictureSequenceDiagram.puml

```diff
@@ -0,0 +1,78 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":ContactListParser" as ContactListParser LOGIC_COLOR
+participant ":PictureCommandParser" as PictureCommandParser LOGIC_COLOR
+participant "p:PictureCommand" as PictureCommand LOGIC_COLOR
+end box
+
+box UI UI_COLOR_T1
+participant ":MainWindow" as MainWindow UI_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute(picture INDEX)
+activate LogicManager
+
+LogicManager -> ContactListParser : parseCommand(picture INDEX))
+activate ContactListParser
+
+create PictureCommandParser
+ContactListParser -> PictureCommandParser
+activate PictureCommandParser
+
+create PictureCommand
+PictureCommandParser -> PictureCommand : parse(INDEX)
+activate PictureCommand
+PictureCommand --> PictureCommandParser :
+deactivate PictureCommand
+
+PictureCommandParser --> ContactListParser : p
+deactivate PictureCommandParser
+deactivate PictureCommand
+
+ContactListParser --> LogicManager : p
+deactivate ContactListParser
+destroy PictureCommandParser
+
+LogicManager -> PictureCommand : execute()
+activate PictureCommand
+
+PictureCommand -> PictureCommand : getPic()
+activate PictureCommand
+
+PictureCommand -> MainWindow : getFile()
+activate MainWindow
+
+MainWindow --> PictureCommand : file
+deactivate MainWindow
+
+PictureCommand --> PictureCommand : file
+deactivate PictureCommand
+
+PictureCommand -> Model : setPerson
+activate Model
+
+Model --> PictureCommand :
+deactivate Model
+
+PictureCommand -> Model : updateFilteredPersonList
```

Comment:
Same thing here. Might wanna add brackets at the end to explicitly show that it's a method.